### ISSUE TITLE
Does spark.read. JDBC support reading bitmap fields from clickHouse tables？？

### ISSUE THREAD

**Issue by czk97** [ClickHouse/clickhouse-jdbc#775]
Thanks！

**Comment by zhicwu** [comment]
Yes. Did you check the examples I mentioned in the other thread?

**Comment by czk97** [comment]
> Yes. Did you check the examples I mentioned in the other thread?
Excuse me, which post? I can't find it. Can you tell me？ Thanks

**Comment by zhicwu** [comment]
I meant #773 which was closed by you earlier. As to examples, please refer to [here](../issues/661#issuecomment-985222705).

**Comment by czk97** [comment]
> 



> I meant #773 which was closed by you earlier. As to examples, please refer to [here](../issues/661#issuecomment-985222705).

抱歉哈，我不太明白。我现在想基于spark通过jdbc读取clickhouse bitmap字段，这个貌似需要自定义方言是吗？直接通过spark.read.jdbc会报类型不支持的异常。 示例中没有基于spark读取bitmap字段。
